File: Algorithm/BOJ/Python/Number Theory/1929.py
import math

def find_prime(M, N):
    
    for number in range(M, N + 1):
        if number == 1:
            continue
        if number != 2 and number % 2 == 0:
            continue
        
        max_check_num = int(math.sqrt(number)) + 1
        for i in range(1, max_check_num):
            if i != 1 and number % i == 0:
                break
        else:
            print(number)


M, N = map(int, input().split())
find_prime(M, N)

File: Algorithm/BOJ/C++/Number Theory/1929.cpp
#include <iostream>
#include <math.h>

using namespace std;

void find_prime(int M, int N) {

	for (int number = M; number < N + 2; number++) {
		if (number == 1) {
			continue;
		}

		int max_check_num = int(sqrt(number)) + 1;
		int i = 1;
		while (i < max_check_num) {			
			if ((i != 1) && (number % i == 0)) {
				break;
			}
			i++;
		}
		if (i == max_check_num) {
			cout << number << endl;
		}
	}
}


int main() {
	int M, N;
	cin >> M >> N;

	find_prime(M, N);
	
	return 0;
}
